main.go

```diff
@@ -22,9 +27,28 @@ var (
 )
 
 func main() {
+	var dbHost string;
```

Comment:
I understand that we want to make progress, so sorry for pushing to hard. Yet, config files should not be discouraged only because we containerize tings. It should be easy to do something like this:

`docker run -it -v `$PWD`:/source almighty-core alm /source/config.yml`

The only drawback is that you have to mount in a volume.